debian/changelog

```diff
@@ -1,3 +1,7 @@
+
+  [Marco Pagani]
+  * Added a more flexible version of the GC2 implementation
+
```

Comment:
Perhaps you should mention that the results are now different with respect to engine-3.13.